interface/wx/window.h

```diff
@@ -4099,6 +4099,19 @@ class wxWindow : public wxEvtHandler
     */
     virtual void UpdateWindowUI(long flags = wxUPDATE_UI_NONE);
 
+    /**
+        When UpdateWindowUI() runs, it creates instances of
+        wxUpdateUIEvent.  Those instances should vary depending on the
+        window that the wxUpdateUIEvent will control.  For example, a
+        wxCheckBox with wxCHK_3STATE should enable
+        wxUpdateUIEvent::Is3State(), but most other windows should not.
+
```

Comment:
This seems a bit unfinished to me, I'd add

```suggestion
This function can be overridden to perform window-specific initializations,
such as enabling setting the checkable state.
```

Reply:
OK